Zend/zend_types.h

```diff
@@ -1274,6 +1274,7 @@ static zend_always_inline uint32_t zval_delref_p(zval* pz) {
  * the Z_EXTRA space when copying property default values etc. We define separate
  * macros for this purpose, so this workaround is easier to remove in the future. */
 #define IS_PROP_UNINIT 1
+#define IS_PROP_REINITABLE 2  /* It has impact only on readonly properties */
```

Comment:
Nit: Using a bit-shift here would make it a little bit more obvious that these are flags.

Reply:
OK, makes sense to me as well